plugins/source/postgresql/client/client.go

```diff
@@ -16,15 +16,15 @@ import (
 )
 
 type Client struct {
+	plugin.UnimplementedDestination
 	Conn                *pgxpool.Pool
 	logger              zerolog.Logger
-	spec                specs.Source
-	metrics             *source.Metrics
 	pluginSpec          Spec
 	currentDatabaseName string
 	currentSchemaName   string
 	pgType              pgType
-	Tables              schema.Tables
+	tables              schema.Tables
+	cdcId               string
```

Comment:
Since plugins don't have access to the top level spec we can't use the source name as the replication name like before. I added a plugin level configuration for now, but we should figure this out before merging this PR as this is a breaking change